ls1intum/Artemis
# `Programming exercises`: Improve information for changing auxiliary repositories

### Checklist
#### General


#### Client

### Motivation and Context


#5551 mentioned a potential bug that is currently the intended feature. But the warning on Artemis did not 100% reflect the behavior.
The intended behavior is that all changes afterwards need to be carried out manually in the VCS and CIS.

### Description

This PR enhances the information about changing auxiliary repositories after the creation of a programming exercise.
Additionally a warning is now shown after editing a programming exercise if the auxiliary repositories were edited.

### Steps for Testing

Prerequisites:

1. Check the translation keys
2. Create an exercise with an auxiliary repository (no warning should be shown when saving)
3. Edit that exercise and change (add, remove, edit) an auxiliary repository and check that a warning pops up on the top right when saving

### Review Progress





#### Code Review
#### Manual Tests
